index.html

```diff
@@ -106,11 +113,13 @@ <h5 class="card1__text-item2">2015</h5>
                 <div class="buttons__technology buttons__languages-text">css</div>
                 <div class="buttons__technology buttons__languages-text">javaScript</div>
               </div>
-              <a class="buttons__main buttons__text" href="#">See Project</a>
+              <a class="buttons__main buttons__text proj2 see-pro2" href="#">See Project</a>
             </div>
           </div>
```

Comment:
- [ ] Please make sure that your code is responsive 

![image](https://user-images.githubusercontent.com/74744787/181924554-96bed83e-75d6-4a5b-ac98-4b44e8d0c541.png)
![image](https://user-images.githubusercontent.com/74744787/181924572-a35ebb7c-3408-4b4e-baa6-3516f6880865.png)